multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt

```diff
@@ -56,7 +56,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("KomikIndo.co", "https://komikindo.co", "id", className = "KomikindoCo", overrideVersionCode = 3),
         SingleLang("KomikMama", "https://komikmama.co", "id", overrideVersionCode = 1),
         SingleLang("KomikManhwa", "https://komikmanhwa.me", "id", isNsfw = true),
-        SingleLang("KumaPoi", "https://kumapoi.me", "id", isNsfw = true),
+        SingleLang("KumaPoi", "https://kumapoi.club/", "id", isNsfw = true, overrideVersionCode = 2),
```

Comment:
Remove the trailing slash.

```suggestion
        SingleLang("KumaPoi", "https://kumapoi.club", "id", isNsfw = true, overrideVersionCode = 2),
```